src/ztc/api/tests/test_eigenschap.py

```diff
@@ -40,21 +39,11 @@ def setUp(self):
             referentie_naar_eigenschap=referentie
         )
 
-        self.eigenschap_list_url = reverse('api:eigenschap-list', kwargs={
-            'version': self.API_VERSION,
-            'catalogus_pk': self.catalogus.pk,
-            'zaaktype_pk': self.zaaktype.pk
-        })
+        self.eigenschap_list_url = reverse('api:eigenschap-list')
```

Comment:
the `api:` namespace was dropped, this shouldn't work but the tests are disabled obviously :-)

Reply:
Yeah, I'll update and turn on these tests when I change resources to writable.